changelog/README.md

```diff
@@ -0,0 +1,79 @@
+
+To follow this strategy you should create a `.md` file for every PR. Choose a directory:
+
+- `changelog/unreleased/features` for **Features** changes
+- `changelog/unreleased/bugfixes` for **Bug fixes and improvements** changes
+- `changelog/unreleased/issues` for **Known issues :warning:** changes
+- `changelog/unreleased/other` for other changes
+
+You can use anything that allow .md format in changelog files.
+
+If you have implemented several features or bugfixes you should describe all of them:
+
+```
+- Description of changes in md format
+- Description of changes in md format also
+```
+
+You can choose any name for your changelog files because the GitHub action will rename files in
+`changelog/unreleased/features` and `changelog/unreleased/bugfixes` directories to `${PR_NUMBER}.md` when you open a PR.
+
+Every push to the main or release branch Assemble changelog GitHub action will be executed:
+
+* collect all files from `changelog/unreleased`
+* assemble the changelog like:
+
+```
+#### Features
+- Feature 1 [#1234](https://github.com/mapbox/mapbox-navigation-android/pull/1234)
+- Feature 2 [#2345](https://github.com/mapbox/mapbox-navigation-android/pull/2345)
+
+#### Bug fixes and improvements
+- Bugfix 3 [#3456](https://github.com/mapbox/mapbox-navigation-android/pull/3456)
+- Bugfix 4 [#4567](https://github.com/mapbox/mapbox-navigation-android/pull/4567)
+
+#### Known issues :warning:
+- Issue 1
+- Issue 2
+
+Some other changes
+```
+
+* write the changelog to the `changelog/unreleased/CHANGELOG.md` file
+
+Every release the release train app will:
+
+* get changelog from `changelog/unreleased/CHANGELOG.md` file
+* add information about dependencies and compile changelog like:
+```
+## Mapbox Navigation SDK 1.1.1 - 13 December, 2022
+### Changelog
+[Changes between v1.1.0 and v1.1.1](https://github.com/mapbox/mapbox-navigation-android/compare/v1.1.0...v1.1.1)
+
+#### Features
+- Feature 1 [#1234](https://github.com/mapbox/mapbox-navigation-android/pull/1234)
+- Feature 2 [#2345](https://github.com/mapbox/mapbox-navigation-android/pull/2345)
+
+#### Bug fixes and improvements
+- Bugfix 3 [#3456](https://github.com/mapbox/mapbox-navigation-android/pull/3456)
+- Bugfix 4 [#4567](https://github.com/mapbox/mapbox-navigation-android/pull/4567)
+
+#### Known issues :warning:
+- Issue 1
+- Issue 2
+
+Some other changes
+
+### Mapbox dependencies
+This release depends on, and has been tested with, the following Mapbox dependencies:
+- Mapbox Maps SDK `v10.8.0` ([release notes](https://github.com/mapbox/mapbox-maps-android/releases/tag/v10.8.0))
+- Mapbox Navigation Native `v115.0.1`
+- Mapbox Core Common `v23.0.0`
+- Mapbox Java `v6.8.0` ([release notes](https://github.com/mapbox/mapbox-java/releases/tag/v6.8.0))
+- Mapbox Android Core `v5.0.2` ([release notes](https://github.com/mapbox/mapbox-events-android/releases/tag/core-5.0.2))
+```
+* add compiled changelog to `CHANGELOG.md` file
+* delete all files in `changelog/unreleased` dir
```

Comment:
@SevaZhukov will this system work with the android auto changelog https://github.com/mapbox/mapbox-navigation-android/blob/main/libnavui-androidauto/CHANGELOG.md

All good to consider it out of scope but it would be nice if it supported multiple release channels